src/FuseDigital.QuickSetup.Domain/Entities/EntityFoundException.cs

```diff
@@ -0,0 +1,78 @@
+using Volo.Abp;
+using Volo.Abp.Domain.Entities;
+
+namespace FuseDigital.QuickSetup.Entities;
+
+public class EntityFoundException : AbpException
```

Comment:
Can I suggest that it is rather called `ExistingEntityFoundException`? At first glance, I thought it was a typo. Another option might be `EntityAlreadyExistsException`?